yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -116,32 +80,15 @@ type tableMigStatusOutputRow struct {
 	totalCount         int64
 	importedCount      int64
 	percentageComplete float64
-	totalEvents        int64
-	numInserts         int64
-	numUpdates         int64
-	numDeletes         int64
-	finalRowCount      int64
 }
 
 // Note that the `import data status` is running in a separate process. It won't have access to the in-memory state
 // held in the main `import data` process.
-func runImportDataStatusCmd(tgtconf *tgtdb.TargetConf, streamChanges bool) error {
+func runImportDataStatusCmd(tgtconf *tgtdb.TargetConf) error {
```

Comment:
We don't seem to be using tgtconf anywhere.. because now we don't connect to target. If so, we can remove this arg

Reply:
Oh yes, missed it, thanks will remove.